d3a_interface/schemas.py

```diff
@@ -121,6 +122,27 @@ class ScenarioSchemas:
                                                      {'type': 'string'}]}
                 }
             },
+            "home_meter": {
+                "type": "object",
+                "properties": {
+                    "name": {"type": "string"},
+                    "type": {"enum": ["HomeMeter"]},
+                    "number_of_clones": {"type": "number"},
+                    "uuid": {"type": "string"},
+                    "libraryUUID": {"anyOf": [{"type": "string"}, {"type": "null"}]},
+                    "initial_selling_rate": {"anyOf": [{"type": "number"}, {"type": "null"}]},
+                    "final_selling_rate": {"type": "number"},
+                    "initial_buying_rate": {"type": "number"},
+                    "final_buying_rate": {"anyOf": [{"type": "number"}, {"type": "null"}]},
```

Comment:
Here I followed the same schema that was already defined for `pv` and `storage` devices. I didn't want to change it because I wasn't sure about the reason why it was defined that way. Could you please give me some info about it?

Reply:
Maybe that's because `initial_selling_rate` and `final_buying_rate` can be `None`, in case `use_market_maker_rate` is `True`.

Reply:
I think you got it right Pierpaolo. In case `use_market_maker_rate = True` , `final_buying_rate` and  `initial_selling_rate` could be None. The others should never be None.